### PR TITLE
Users page linked to be

### LINKED ISSUE #4: Connect to BE (API Calls)
http://xed.im:5000/docs/index.html

### PR DESCRIPTION
**Pull request by filippo-monteleone**
close #4

### CODE REVIEW

File: src/routes/(layout)/user/+page.svelte [hunk 1]
```diff
@@ -1,21 +1,39 @@
...
+		let months = Math.floor(days / 30);
+		let years = Math.floor(months / 12);
+
+		if (years > 0) {
+			return years == 1 ? `${years} year ago` : `${years} years ago`;
+		} else if (months > 0) {
+			return months == 1 ? `${months} month ago` : `${months} moths ago`;
+		}
+
+		return days == 1 ? `${days} day ago` : `${days} days ago`;
```
Comment (AgeOfWar) [inline src/routes/(layout)/user/+page.svelte]: This could be done with dayjs plugins duration + relativeTime

**Review by AgeOfWar — APPROVED** [review]

File: src/routes/(layout)/user/+page.svelte (hunk 1 above)
Comment (xedom) [reply src/routes/(layout)/user/+page.svelte]: For now i would only add relativeTime

**Review by xedom — CHANGES_REQUESTED** [review]
add relativeTime

File: src/lib/backend.ts
Comment (xedom) [inline src/lib/backend.ts:1]: no bueno

File: src/routes/(layout)/user/+page.ts
Comment (xedom) [inline src/routes/(layout)/user/+page.ts:1]: huh

**Review by xedom — CHANGES_REQUESTED** [review]
fix

**Review by xedom — APPROVED** [review]

**Review by AgeOfWar — APPROVED** [review]

**Pull request merged**